client/package.json

```diff
@@ -10,6 +10,7 @@
     "start": "yarn start-remote",
     "build": "lerna run --scope @openmsupply-client/* build",
     "build-stats": "lerna run --scope @openmsupply-client/* build-stats",
+    "build-plugins": "lerna run --scope @openmsupply-client/* build-plugin",
```

Comment:
not strictly required, it allows all plugins to be built together and means that you can build them from the client dir